### PR TITLE
Kth minimum

### PR DESCRIPTION
**Pull request by Amit366**
Added the code for kth minimum element in array.

### CODE REVIEW

**Comment by Amit366** [comment]
@Rohan27s please review

**Comment by Rohan27s** [comment]
mention the issue first

**Comment by Amit366** [comment]
Fixes: #162

File: Code/Python/Kth_Minimum_in_array.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    arr = []
+    n = int(input("Enter number of elements: "))
+    for i in range(n):
+        element = int(input("Enter element: "))
+        arr.append(element)
+
+    k = int(input("Enter the number k: "))
+    print("K'th smallest element is",
+          kthSmallest(arr, n, k))
+
```
Comment (Rohan27s) [inline Code/Python/Kth_Minimum_in_array.py:23]: Mention two test cases and Time & Space complexity

**Review by Rohan27s — APPROVED** [review]

**Comment by Amit366** [comment]
@Rohan27s done

**Comment by Amit366** [comment]
@tarun26091999 please review sir

**Review by Rohan27s — CHANGES_REQUESTED** [review]
Don't commit multiple files in a single Pr

**Pull request closed without merging**